source/core_sntp_client.c

```diff
@@ -80,8 +76,8 @@ SntpStatus_t Sntp_Init( SntpContext_t * pContext,
     {
         LogError( ( "Invalid parameter: Function members of passed authentication interface "
                     "cannot be NULL: generateClientAuth=%p, validateServerAuth=%p",
-                    ( void * ) pTransportIntf->recvFrom,
-                    ( void * ) pTransportIntf->sendTo ) );
+                    ( void * ) pAuthIntf->generateClientAuth,
+                    ( void * ) pAuthIntf->validateServerAuth ) );
```

Comment:
These also look like function pointers, so won't they have the same GCC warnings?

Reply:
Oops, forgot about it, fixed.